modules/security-hosts-vms-openshift.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-hosts-vms.adoc
+
+[id="security-hosts-vms-openshift_{context}"]
+= Securing {product-title}
+When you deploy {product-title}, you have the choice of an
+installer-provisioned infrastructure (there are several available platforms
+or your own user-provisioned infrastructure.
+Some low-level security-related configuration, such as enabling FIPS
+compliance or adding kernel modules required at first boot, might 
+benefit from a user-provisioned infrastructure.
+Likewise, user-provisioned infrastructure is appropriate for disconnected (air gap) {product-title} deployments.
+
+Keep in mind that, when it comes to making security enhancements and other
+configuration changes to {product-title}, the goals should include:
+
+* Keeping the underlying nodes as generic as possible. You want to be able to
+easily throw away and spin up similar nodes quickly and in prescriptive ways.
+* Managing modifications to nodes through {product-title} as much as possible,
+rather than making direct, one-off changes to the nodes.
+
+In pursuit of those goals, most node changes should be done during installation through Ignition
+or later using Machineconfigs that are applied to sets of nodes by the Machine Config Operator.
```

Comment:
Machineconfigs -> MachineConfigs

A few other places in the PR

Reply:
Got it. Thanks. Done.